acceptance: TestDockerCLI failed

[(acceptance).TestDockerCLI failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2765590&tab=buildLog) on [master@fc68d927f946103a757888fcbd3ac20904c57a6b](https://github.com/cockroachdb/cockroach/commits/fc68d927f946103a757888fcbd3ac20904c57a6b):

```
=== RUN   TestDockerCLI
    test_log_scope.go:73: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestDockerCLI325791061
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestDockerCLI
    cli_test.go:93: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestDockerCLI325791061
--- FAIL: TestDockerCLI (408.87s)
=== RUN   TestDockerCLI/test_demo_node_cmds.tcl
=== CONT  TestDockerCLI/test_demo_node_cmds.tcl
    cli_test.go:89: non-zero exit code: 1
    --- FAIL: TestDockerCLI/test_demo_node_cmds.tcl (37.57s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestDockerCLI PKG=./pkg/acceptance TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestDockerCLI.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>


Jira issue: CRDB-2727

[(acceptance).TestDockerCLI failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2790309&tab=buildLog) on [master@fc2f2c36f3204ad75e76fc52f2fdfd47e9a8b161](https://github.com/cockroachdb/cockroach/commits/fc2f2c36f3204ad75e76fc52f2fdfd47e9a8b161):

```
=== RUN   TestDockerCLI
    test_log_scope.go:73: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestDockerCLI573492571
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestDockerCLI
    cli_test.go:93: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestDockerCLI573492571
--- FAIL: TestDockerCLI (424.59s)
=== RUN   TestDockerCLI/test_demo_node_cmds.tcl
=== CONT  TestDockerCLI/test_demo_node_cmds.tcl
    cli_test.go:89: non-zero exit code: 1
    --- FAIL: TestDockerCLI/test_demo_node_cmds.tcl (37.66s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestDockerCLI PKG=./pkg/acceptance TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #61896 acceptance: TestDockerCLI failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-21.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-21.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestDockerCLI.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(acceptance).TestDockerCLI failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2791242&tab=buildLog) on [master@75cf42e4964137196713bcd539806f367883f52c](https://github.com/cockroachdb/cockroach/commits/75cf42e4964137196713bcd539806f367883f52c):

```
=== RUN   TestDockerCLI
    test_log_scope.go:73: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestDockerCLI372985801
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestDockerCLI
    cli_test.go:93: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestDockerCLI372985801
--- FAIL: TestDockerCLI (413.22s)
=== RUN   TestDockerCLI/test_demo_node_cmds.tcl
=== CONT  TestDockerCLI/test_demo_node_cmds.tcl
    cli_test.go:89: non-zero exit code: 1
    --- FAIL: TestDockerCLI/test_demo_node_cmds.tcl (37.20s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestDockerCLI PKG=./pkg/acceptance TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #61896 acceptance: TestDockerCLI failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-21.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-21.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestDockerCLI.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

acceptance.TestDockerCLI [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3229527&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3229527&tab=artifacts#/) on master @ [eb6b1184d061dae78ba9f4bbe74d35b18ce7738a](https://github.com/cockroachdb/cockroach/commits/eb6b1184d061dae78ba9f4bbe74d35b18ce7738a):


```
=== RUN   TestDockerCLI
    test_log_scope.go:73: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestDockerCLI587625740
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestDockerCLI
    cli_test.go:93: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestDockerCLI587625740
--- FAIL: TestDockerCLI (395.08s)
=== RUN   TestDockerCLI/test_demo_node_cmds.tcl
=== CONT  TestDockerCLI/test_demo_node_cmds.tcl
    cli_test.go:89: non-zero exit code: 1
    --- FAIL: TestDockerCLI/test_demo_node_cmds.tcl (38.27s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestDockerCLI PKG=./pkg/acceptance TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
<p>Parameters in this failure:

- GOFLAGS=-json
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #61896 acceptance: TestDockerCLI failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
tamird marked as alumn{us/a}; resolving to tbg instead
```
</p>
</details>

/cc @cockroachdb/sql-experience tbg
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

acceptance.TestDockerCLI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_Tests_Acceptance/8592742?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_Tests_Acceptance/8592742?buildTab=artifacts#/) on master @ [8e24570fa366ed038c6ae65f50db5d8e22826db0](https://github.com/cockroachdb/cockroach/commits/8e24570fa366ed038c6ae65f50db5d8e22826db0):


```
=== RUN   TestDockerCLI
    test_log_scope.go:161: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/b82317f51c7daead0d1312a2f3300dbe/logTestDockerCLI1420520336
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestDockerCLI
    cli_test.go:95: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/b82317f51c7daead0d1312a2f3300dbe/logTestDockerCLI1420520336
--- FAIL: TestDockerCLI (1136.04s)
=== RUN   TestDockerCLI/test_demo_partitioning.tcl
=== CONT  TestDockerCLI/test_demo_partitioning.tcl
    cli_test.go:91: non-zero exit code: 1
    --- FAIL: TestDockerCLI/test_demo_partitioning.tcl (351.46s)
```
<p>Parameters: <code>GOFLAGS=-json</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #77886 acceptance: TestDockerCLI failed [C-test-failure O-robot T-server-and-security branch-release-22.1]
- #61896 acceptance: TestDockerCLI failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

acceptance.TestDockerCLI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/8902977?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/8902977?buildTab=artifacts#/) on master @ [70724583f525ee83caff4be14474aa87e47f42f4](https://github.com/cockroachdb/cockroach/commits/70724583f525ee83caff4be14474aa87e47f42f4):


```
=== RUN   TestDockerCLI
    test_log_scope.go:161: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/cae8acf78e91f72c41610d0ec1f275da/logTestDockerCLI2889441075
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestDockerCLI
    cli_test.go:95: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/cae8acf78e91f72c41610d0ec1f275da/logTestDockerCLI2889441075
--- FAIL: TestDockerCLI (854.30s)
=== RUN   TestDockerCLI/test_demo_node_cmds.tcl
=== CONT  TestDockerCLI/test_demo_node_cmds.tcl
    cli_test.go:91: non-zero exit code: 1
    --- FAIL: TestDockerCLI/test_demo_node_cmds.tcl (14.78s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #77886 acceptance: TestDockerCLI failed [C-test-failure O-robot T-server-and-security branch-release-22.1]
- #61896 acceptance: TestDockerCLI failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

acceptance.TestDockerCLI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/9128467?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/9128467?buildTab=artifacts#/) on master @ [b791abd220ad0afa9a1815d44e3613ee8097936a](https://github.com/cockroachdb/cockroach/commits/b791abd220ad0afa9a1815d44e3613ee8097936a):


```
=== RUN   TestDockerCLI
    test_log_scope.go:161: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/cae8acf78e91f72c41610d0ec1f275da/logTestDockerCLI2838542618
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestDockerCLI
    cli_test.go:95: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/cae8acf78e91f72c41610d0ec1f275da/logTestDockerCLI2838542618
--- FAIL: TestDockerCLI (944.92s)
=== RUN   TestDockerCLI/test_copy.tcl
=== CONT  TestDockerCLI/test_copy.tcl
    cli_test.go:91: non-zero exit code: 1
    --- FAIL: TestDockerCLI/test_copy.tcl (35.46s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #77886 acceptance: TestDockerCLI failed [C-test-failure O-robot T-server-and-security branch-release-22.1]
- #61896 acceptance: TestDockerCLI failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

acceptance.TestDockerCLI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_Tests_Acceptance/9136927?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_Tests_Acceptance/9136927?buildTab=artifacts#/) on master @ [53dbb86acb1d48309530181b94838faf937084d3](https://github.com/cockroachdb/cockroach/commits/53dbb86acb1d48309530181b94838faf937084d3):


```
=== RUN   TestDockerCLI
    test_log_scope.go:161: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/b82317f51c7daead0d1312a2f3300dbe/logTestDockerCLI1678802933
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestDockerCLI
    cli_test.go:95: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/b82317f51c7daead0d1312a2f3300dbe/logTestDockerCLI1678802933
--- FAIL: TestDockerCLI (718.41s)
=== RUN   TestDockerCLI/test_demo_global_insecure.tcl
=== CONT  TestDockerCLI/test_demo_global_insecure.tcl
    cli_test.go:91: non-zero exit code: 1
    --- FAIL: TestDockerCLI/test_demo_global_insecure.tcl (92.46s)
```
<p>Parameters: <code>GOFLAGS=-json</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #77886 acceptance: TestDockerCLI failed [C-test-failure O-robot T-server-and-security branch-release-22.1]
- #61896 acceptance: TestDockerCLI failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

acceptance.TestDockerCLI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/9138149?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/9138149?buildTab=artifacts#/) on master @ [1cf75095871a9df935c036c674783dca73a02134](https://github.com/cockroachdb/cockroach/commits/1cf75095871a9df935c036c674783dca73a02134):


```
=== RUN   TestDockerCLI
    test_log_scope.go:161: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/cae8acf78e91f72c41610d0ec1f275da/logTestDockerCLI4290031421
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestDockerCLI
    cli_test.go:95: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/cae8acf78e91f72c41610d0ec1f275da/logTestDockerCLI4290031421
--- FAIL: TestDockerCLI (896.33s)
=== RUN   TestDockerCLI/test_multiline_statements_libedit.tcl
=== CONT  TestDockerCLI/test_multiline_statements_libedit.tcl
    cli_test.go:91: non-zero exit code: 1
    --- FAIL: TestDockerCLI/test_multiline_statements_libedit.tcl (35.13s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #77886 acceptance: TestDockerCLI failed [C-test-failure O-robot T-server-and-security branch-release-22.1]
- #61896 acceptance: TestDockerCLI failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

acceptance.TestDockerCLI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/9138998?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/9138998?buildTab=artifacts#/) on master @ [9aa4f3485c75397e9ce0edabbf22c6343e1a901a](https://github.com/cockroachdb/cockroach/commits/9aa4f3485c75397e9ce0edabbf22c6343e1a901a):


```
=== RUN   TestDockerCLI
    test_log_scope.go:161: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/cae8acf78e91f72c41610d0ec1f275da/logTestDockerCLI284892776
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestDockerCLI
    cli_test.go:95: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/cae8acf78e91f72c41610d0ec1f275da/logTestDockerCLI284892776
--- FAIL: TestDockerCLI (981.62s)
=== RUN   TestDockerCLI/test_disable_replication.tcl
=== CONT  TestDockerCLI/test_disable_replication.tcl
    cli_test.go:91: non-zero exit code: 1
    --- FAIL: TestDockerCLI/test_disable_replication.tcl (45.89s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #77886 acceptance: TestDockerCLI failed [C-test-failure O-robot T-server-and-security branch-release-22.1]
- #61896 acceptance: TestDockerCLI failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

acceptance.TestDockerCLI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/9139943?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/9139943?buildTab=artifacts#/) on master @ [139427b0f2218bb7dbae86cbdbfde6f38850264c](https://github.com/cockroachdb/cockroach/commits/139427b0f2218bb7dbae86cbdbfde6f38850264c):


```
=== RUN   TestDockerCLI
    test_log_scope.go:161: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/cae8acf78e91f72c41610d0ec1f275da/logTestDockerCLI1679803517
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestDockerCLI
    cli_test.go:95: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/cae8acf78e91f72c41610d0ec1f275da/logTestDockerCLI1679803517
--- FAIL: TestDockerCLI (883.88s)
=== RUN   TestDockerCLI/test_demo_global.tcl
=== CONT  TestDockerCLI/test_demo_global.tcl
    cli_test.go:91: non-zero exit code: 1
    --- FAIL: TestDockerCLI/test_demo_global.tcl (92.95s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #77886 acceptance: TestDockerCLI failed [C-test-failure O-robot T-server-and-security branch-release-22.1]
- #61896 acceptance: TestDockerCLI failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

acceptance.TestDockerCLI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/9365990?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/9365990?buildTab=artifacts#/) on master @ [2bc171df31dd61e622839e06389082bcb229af70](https://github.com/cockroachdb/cockroach/commits/2bc171df31dd61e622839e06389082bcb229af70):


```
=== RUN   TestDockerCLI
    test_log_scope.go:161: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/cae8acf78e91f72c41610d0ec1f275da/logTestDockerCLI2396787071
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestDockerCLI
    cli_test.go:95: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/cae8acf78e91f72c41610d0ec1f275da/logTestDockerCLI2396787071
--- FAIL: TestDockerCLI (1002.51s)
=== RUN   TestDockerCLI/test_demo_memory_warning.tcl
=== CONT  TestDockerCLI/test_demo_memory_warning.tcl
    cli_test.go:91: non-zero exit code: 1
    --- FAIL: TestDockerCLI/test_demo_memory_warning.tcl (33.35s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #100319 acceptance: TestDockerCLI failed [C-test-failure O-robot branch-release-23.1]
- #77886 acceptance: TestDockerCLI failed [C-test-failure O-robot T-server-and-security branch-release-22.1]
- #61896 acceptance: TestDockerCLI failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

acceptance.TestDockerCLI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/9372472?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/9372472?buildTab=artifacts#/) on master @ [6ee3a0d1629de554dd101f39bb8d1911ba95a032](https://github.com/cockroachdb/cockroach/commits/6ee3a0d1629de554dd101f39bb8d1911ba95a032):


```
=== RUN   TestDockerCLI
    test_log_scope.go:161: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/cae8acf78e91f72c41610d0ec1f275da/logTestDockerCLI2124980660
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestDockerCLI
    cli_test.go:95: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/cae8acf78e91f72c41610d0ec1f275da/logTestDockerCLI2124980660
--- FAIL: TestDockerCLI (898.97s)
=== RUN   TestDockerCLI/test_demo_memory_warning.tcl
=== CONT  TestDockerCLI/test_demo_memory_warning.tcl
    cli_test.go:91: non-zero exit code: 1
    --- FAIL: TestDockerCLI/test_demo_memory_warning.tcl (33.56s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #100319 acceptance: TestDockerCLI failed [C-test-failure O-robot branch-release-23.1]
- #77886 acceptance: TestDockerCLI failed [C-test-failure O-robot T-server-and-security branch-release-22.1]
- #61896 acceptance: TestDockerCLI failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

acceptance.TestDockerCLI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/9402124?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/9402124?buildTab=artifacts#/) on master @ [db4c5acd3a21a255a39d3dd657eac75c1b0783ad](https://github.com/cockroachdb/cockroach/commits/db4c5acd3a21a255a39d3dd657eac75c1b0783ad):


```
=== RUN   TestDockerCLI
    test_log_scope.go:161: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/cae8acf78e91f72c41610d0ec1f275da/logTestDockerCLI4176480291
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestDockerCLI
    cli_test.go:95: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/cae8acf78e91f72c41610d0ec1f275da/logTestDockerCLI4176480291
--- FAIL: TestDockerCLI (874.87s)
=== RUN   TestDockerCLI/test_demo_memory_warning.tcl
=== CONT  TestDockerCLI/test_demo_memory_warning.tcl
    cli_test.go:91: non-zero exit code: 1
    --- FAIL: TestDockerCLI/test_demo_memory_warning.tcl (33.23s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #100319 acceptance: TestDockerCLI failed [C-test-failure O-robot branch-release-23.1]
- #77886 acceptance: TestDockerCLI failed [C-test-failure O-robot T-server-and-security branch-release-22.1]
- #61896 acceptance: TestDockerCLI failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

acceptance.TestDockerCLI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/9402493?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/9402493?buildTab=artifacts#/) on master @ [c529fca30a707bf11974ffa6efed20f7eb69313c](https://github.com/cockroachdb/cockroach/commits/c529fca30a707bf11974ffa6efed20f7eb69313c):


```
=== RUN   TestDockerCLI
    test_log_scope.go:161: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/cae8acf78e91f72c41610d0ec1f275da/logTestDockerCLI2920120574
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestDockerCLI
    cli_test.go:95: -- test log scope end --
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/_tmp/cae8acf78e91f72c41610d0ec1f275da/logTestDockerCLI2920120574
--- FAIL: TestDockerCLI (820.32s)
=== RUN   TestDockerCLI/test_demo_memory_warning.tcl
=== CONT  TestDockerCLI/test_demo_memory_warning.tcl
    cli_test.go:91: non-zero exit code: 1
    --- FAIL: TestDockerCLI/test_demo_memory_warning.tcl (33.53s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #100319 acceptance: TestDockerCLI failed [C-test-failure O-robot branch-release-23.1]
- #77886 acceptance: TestDockerCLI failed [C-test-failure O-robot T-server-and-security branch-release-22.1]
- #61896 acceptance: TestDockerCLI failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDockerCLI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/cli/interactive_tests/test_demo_memory_warning.tcl
+++ b/pkg/cli/interactive_tests/test_demo_memory_warning.tcl
@@ -7,7 +7,11 @@ spawn $argv demo --no-line-editor --no-example-database --max-sql-memory=10% --n
 
 eexpect "WARNING: HIGH MEMORY USAGE"
 
-eexpect "defaultdb>"
+# try to exit early via Ctrl+C.
 interrupt
+# If the demo command started the prompt before we got a chance to send Ctrl+C above,
+# it's not going to accept Ctrl+C any more. Instead, send a quit command.
+send "\r\\q\r"
+eexpect eof
 
 end_test

